src/Components/AuthShared/SelectOption/SelectOption.tsx

```diff
@@ -0,0 +1,54 @@
+import { CircleChevronDown, ListFilter } from "lucide-react";
+import React, { ComponentProps } from "react";
+
+type SelectOptionProps = {
+  label: string;
+
+  error?: string | undefined;
+} & ComponentProps<"select">;
+
+const SelectOption = React.forwardRef<HTMLSelectElement, SelectOptionProps>(
+  ({ label, error, ...props }, ref) => {
+    console.log(error);
+    return (
+      <div className="flex flex-col gap-2 mb-3">
+        <label
+          htmlFor={props.id}
+          className="block  text-[#ffff] ms-6 font-bold text-lg"
+        >
+          {label}
+        </label>
+
+        <div className="relative">
+          <select
+            className={` appearance-none focus:ring-0   text-lg ps-[3.7rem] block w-full  border-[3px] focus:outline-none focus:border-[${
+              error ? "#C5D86D" : "#C5D86D"
```

Comment:
Fixed colors are not a good practice to use across the application, this will make it difficult to make changes in the colors in the application.